packages/runner-ct/src/lib/event-manager.js

```diff
@@ -195,6 +196,17 @@ const eventManager = {
       this._clearAllCookies()
     })
 
+    const hmrSocket = connectWebpackHmr({
+      url: `${window.location.origin}/cypress-webpack-hmr-socket`,
+      onReload: () => {
+        if (state.isInitialBuildSucceed) {
```

Comment:
Same remark `isInitialBuildSucessful` or `isInitialBuildOK` would be more english
```suggestion
        if (state.isInitialBuildOK) {
```